core/frontend/src/main/scala/io/udash/routing/UrlChangeProvider.scala

```diff
@@ -5,13 +5,16 @@ import io.udash.core.Url
 import io.udash.properties.MutableBufferRegistration
 import io.udash.utils.Registration
 import org.scalajs.dom
-import org.scalajs.dom.Element
+import org.scalajs.dom.{Element, Location}
 import org.scalajs.dom.raw.HashChangeEvent
 
 import scala.scalajs.js
 
 /** Provides information about current URL. */
 trait UrlChangeProvider {
+  /** Enables all required event listeners. */
+  def initialize(): Unit
```

Comment:
Should I invoke it myself? Is it invoked by the framework? Add more docs please.